.github/actions/backup-and-upload-database/action.yml

```diff
@@ -0,0 +1,70 @@
+name: Backup and Upload production DB
+description: Backup production DB and upload to azure
+
+inputs:
+  azure-credentials:
+    description: Azure credentials
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - name: Set AKS credentials
+      shell: bash
+      run: az aks get-credentials -g s189p01-tsc-pd-rg -n s189p01-tsc-production-aks
+
+    - name: Install kubectl
+      uses: azure/setup-kubectl@v3
+
+    - name: Install konduit
+      shell: bash
+      run: make install-konduit
+
+    - name: Backup database
+      shell: bash
+      run: |
+        bin/konduit.sh npq-registration-production-web -- pg_dump -E utf8 --compress=1 --clean --if-exists --no-privileges --no-owner --verbose -f backup-production.sql.gz
+
+    - name: Set connection string
+      shell: bash
+      run: |
+        STORAGE_CONN_STR=$(az storage account show-connection-string -g s189p01-cpdnpq-pd-rg -n s189p01cpdnpqdbbkppdsa --query 'connectionString')
```

Comment:
The storage account `s189p01cpdnpqdbbkppdsa` doesn't exist yet. @cwrw, did you just create this manually using Azure Portal in ECF?

Reply:
It should be created automatically by https://github.com/DFE-Digital/npq-registration/blob/main/terraform/application/database.tf#L13

Reply:
when we checked however last time in ECF it already existed so this should exist already for NPQ?

Reply:
act nvm I checked and `s189p01cpdnpqdbbkppdsa` already exists so all good

Reply:
Ah, thanks. I looked around the code and couldn't see any reference to it being built!